packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -201,7 +228,8 @@ DropdownButton.propTypes = propTypes;
 DropdownButton.defaultProps = defaultProps;
 DropdownButton.contextType = ThemeContext;
 
-export default DropdownButton;
+export default injectIntl(DropdownButton);
+// export default DropdownButton;
```

Comment:
```suggestion
// export default DropdownButton;
```
this should be removed if no longer required

Reply:
Done.